Makefile

```diff
@@ -5,19 +5,26 @@ ENTER_VENV := source venv/bin/activate
 
 .DEFAULT_GOAL := all
 
+AWS_PROFILE := default
```

Comment:
If we run these the same way we did before, on the ops jenkins with a read only instance level AWS profile, then it'll likely throw an exception for a profile named `default` not existing (since there isn't one in `.aws/credentials`). We ended up switching to `None` (the type not the string `'None'`) to fix the issue.

Something to keep in mind.

Reply:
Absolutely! I changed pytest_wrapper.sh to run `make awsci AWS_PROFILE=$AWS_PROFILE`, so each job should be passing it in.